Week1/homework/app.js

```diff
@@ -1,11 +1,183 @@
-  console.log(bookTitles);
+// --------1.2-------
+// console.log(bookTitles);
+
+// --------1.3------
+function createUl(ul) {
+  for (let i = 0; i < bookTitles.length; i++) {
+    const listTitels = bookTitles[i];
+    const listItems = document.createElement('li');
+    listItems.innerText = listTitels;
+    ul.appendChild(listItems);
+  }
+}
+function main() {
+  const root = document.getElementById('root');
+  const ul = document.createElement('ul');
+  root.appendChild(ul);
+  createUl(ul);
+}
+window.addEventListener('load', main);
+
+// -------1.4-------------
+const bookDetails = {
+  pragmatic_programmer: {
+    title: 'pragmatic programmer',
+    language: 'english',
+    author: 'Andy Hunt',
+    // pragmatic_programmer: './img/pragmatic_programmer.jpg',
+  },
+  clean_code: {
+    title: 'clean code',
+    language: 'english',
+    author: 'Robert C',
+  },
+  code_complete: {
+    title: 'code complete',
+    language: 'english',
+    author: 'Steve McConnel',
+  },
+  design_patterns: {
+    title: 'Design patterns',
+    language: 'english',
+    author: 'Erich Gamma',
+  },
+  Refracting: {
+    title: 'Refracting',
+    language: 'english',
+    author: 'Martin Fowler',
+  },
+  the_mythical_man: {
+    title: 'The Mythical Man',
+    language: 'english',
+    author: 'Fredrick P',
+  },
+  javascript_good_part: {
+    title: 'Javascript good parts',
+    language: 'english',
+    author: 'Douglas Crockford',
+  },
+  structure_and_interpretation_of_computer: {
+    title: 'Structure and interpretation of computer',
+    language: 'english',
+    author: 'Harold Abelson',
+  },
+  c_programming_language: {
+    title: 'C pragramming',
+    language: 'english',
+    author: 'Brian W',
+  },
+  head_first_design: {
+    title: 'Head First Design',
+    language: 'english',
+    author: 'Eric Freeman',
+  },
+};
+console.log(bookDetails);
+
+// ----------1.5----------
+
+function createUiForObject(ul) {
+  for (const key in bookDetails) {
+    const bookTitle = bookDetails[key].title;
+    const bookLanguage = bookDetails[key].language;
+    const bookAuthor = bookDetails[key].author;
+    const titleItems = document.createElement('h2');
+    const languageItems = document.createElement('p');
+    const authorItems = document.createElement('p');
+    const li = document.createElement('li');
+    titleItems.innerText = bookTitle;
+    languageItems.innerText = bookLanguage;
+    authorItems.innerText = bookAuthor;
+    li.appendChild(titleItems);
+    li.appendChild(languageItems);
+    li.appendChild(authorItems);
+    ul.appendChild(li);
+  }
+
+
+function mainObject() {
+  const rootObject = document.getElementById('rootObject');
+  const ul = document.createElement('ul');
+  rootObject.appendChild(ul);
+  createUiForObject(ul);
+}
+window.addEventListener('load', mainObject);
+
+// --------1.6------
+// style.css
+
+// ----------1.7--------
+const images = {
+  // pragmatic_programmer: './img/pragmatic_programmer.jpg',
```

Comment:
Please remove code that you no longer need instead of leaving it in as a comment. It becomes confusing really quickly.

Reply:
I am just keep as a reference for me , to know the other way of linking photos from an image folder.